kickoff/static/kickoff.js

```diff
@@ -33,30 +33,196 @@ function initialSetup() {
+            htmlDescription += '<textarea id="desc' + iDataIndex + '" name="desc' + iDataIndex + '">' + description + '</textarea>';
+            htmlDescription += '<div class="checkbox"><label style="white-space: nowrap; font-weight: bold; text-align: center;" for="isSecurityDriven{{ loop.index }}">';
+            htmlDescription += '<input type="checkbox" ' + (aData.isSecurityDriven ? 'checked' : '') + ' id="isSecurityDriven' + iDataIndex + '"  name="isSecurityDriven' + iDataIndex + '" value="Is Sec Driven?" title="Did we do this release to fix a security issue (chemspill)?"/> Is sec driven?</label></div>';
+            htmlDescription += '<input type="submit" value="Update" class="submitButton" onclick="return updateDesc(\'' + aData.name + '\', ' + iDataIndex + ')" /></div>';
+
+            var tdDescription = $(nRow).find('td')[2];
+            $(tdDescription).html(htmlDescription);
+        },
+        'aoColumns': [
+            {'mData': 'status',
+             'mRender': function(data, type, full) {
+                if (data == 'Complete') {
+                    return '<span class="status_complete">' + data + '</span>';
+                }
+
+                return data;
+            }},
+            {'mData': 'name'},
+            {'mData':  'description'},
+            {'mData': 'submitter'},
+            {'mData': 'submittedAt',
+             'mRender': function(data, type, full) {
+                return '<span style="white-space: nowrap;" class="dateDisplay">' + data + '</span>';
+            }},
+            {'mData': 'branch'},
+            {'mData': 'mozillaRevision'},
+            {'mData': 'mozillaRelbranch'},
+            {'mData': 'commRevision', 'sDefaultContent': 'N/A'},
+            {'mData': 'commRelbranch', 'sDefaultContent': 'N/A'},
+            {'mData': 'dashboardCheck',
+             'mRender': function(data, type, full) {
+                if (data) {
+                    return 'Yes';
+                }
+
+                return 'No';
+            }},
+            {'mData': 'name',
+             'mRender': function(data, type, full) {
+                return '<a style="cursor: pointer; text-decoration: underline;" href="/releases/' + data + '/l10n">Link</a>';
+            }},
+            {'mData': 'partials',
+             'mRender': function(data, type, full) {
+                if (data) {
+                    return data.replace(/,/g,',\n');
+                }
+
+                return '';
+            }},
+            {'mData': 'promptWaitTime',
+             'mRender': function(data, type, full) {
+                if (full.name.contains('Fennec')) {
+                    return 'N/A';
+                } else if (!data) {
+                    return 'Default';
+                }
+
+                return data;
+            }},
+            {'mData': 'comment',
+             'mRender': function(data, type, full) {
+                if (data && data.length > 50) {
```

Comment:
Don't be sorry, this is the principle of code review by peer :) 
Only people who are not doing anything are not making mistake!